hotham/src/components/local_transform.rs

```diff
@@ -76,8 +76,19 @@ impl LocalTransform {
         (self.scale, self.rotation, self.translation) = transform.to_scale_rotation_translation();
     }
 
+    /// Update ONLY rotation and rotation from a [`glam::Affine3A`]
+    pub fn update_rotation_translation_from_affine(&mut self, transform: &glam::Affine3A) {
```

Comment:
This is another halfway solution to having a specific type that only contains rotation and translation.